theories/algebra/ZModP.ec

```diff
@@ -234,6 +234,41 @@ proof. by rewrite -inzmod_mod inzmodB. qed.
 lemma zmodcgrP (i j : int) : zmodcgr i j <=> p %| (i - j).
 proof. by rewrite dvdzE -[0](mod0z p) !eq_inzmod inzmodB subr_eq0. qed.
 
+lemma zmodcgr_trans b a c :
+  zmodcgr a b =>
+  zmodcgr b c =>
+  zmodcgr a c.
+proof. smt(). qed.
```

Comment:
This lemma, and the ones below, fit on a single line. Likewise, the proofs can be compacted